application/services/stripe_client.py

```diff
@@ -33,14 +44,35 @@ def get_created_after(no_of_days_ago):
     return created_after
 
 class StripeClass(object):
+
     def __init__(self, api_key):
         stripe.api_key = api_key
+        self.LOGGER = logger.getLogger(__name__)
+
+    def f(self, func):
+        def stripe_call(*args, **kwargs):
+            if current_user:
```

Comment:
if current_user.is_authenticated:

otherwise, shouldn't you throw an exception the front end should catch and redirect the user to login page. Since we're guarding all app methods with @login_required, it maybe okay to just raise an NotLoggedInException